daemon/containerd/image_delete.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+	"time"
 
+	"github.com/containerd/containerd/errdefs"
```

Comment:
Linter is unhappy:
>  daemon/containerd/image_delete.go:10:2: import "github.com/containerd/containerd/errdefs" imported without alias but must be with alias "cerrdefs" according to config (importas)
	"github.com/containerd/containerd/errdefs"
	^

```suggestion
	cerrdefs "github.com/containerd/containerd/errdefs"
```